web/modules/custom/joinup_core/joinup_core.post_update.php

```diff
@@ -6,3 +6,55 @@
  */
 
 declare(strict_types = 1);
+
+use Drupal\solution\Entity\SolutionInterface;
+
+/**
+ * Remove a selected set of solutions and 1 document from Eurostat.
+ */
+function joinup_core_post_update_0106300(): void {
+  /** @var \Drupal\sparql_entity_storage\SparqlEntityStorageInterface $rdf_storage */
+  $rdf_storage = \Drupal::entityTypeManager()->getStorage('rdf_entity');
+  /** @var \Drupal\collection\Entity\CollectionInterface $collection */
+  $collection = $rdf_storage->load('http://data.europa.eu/w21/01b52000-c0ba-4e64-a32c-79557a743462');
+  $solutions = array_filter($collection->getSolutions(), function (SolutionInterface $solution) {
+    // Remove all solutions starting with the word SCL.
+    if (substr($solution->label(), 0, 4) === 'SCL ') {
+      return TRUE;
+    }
+
+    // Additionally remove 3 specific solutions.
+    return in_array($solution->id(), [
+      // 'General information on the "Standard code lists" project'.
+      'http://ec.europa.eu/eurostat/ramon/miscellaneous/index.cfm?TargetUrl=DSP_GENINFO_SCL',
+      // 'Eurostat SDMX Converter'.
+      'http://data.europa.eu/w21/35ef35c6-a530-4821-8c52-8d796ec86a1d',
+      // 'Eurostat XBRL Reference Architecture'.
+      'http://data.europa.eu/w21/96f74e35-abda-4475-8b2d-22fed2120fc5',
+    ]);
+  });
+
+  $total = count($solutions);
+  $count = 0;
+
+  foreach ($solutions as $solution) {
+    \Drupal::logger('joinup')->notice('[%count/%total] Deleting solution %title', [
+      '%count' => ++$count,
+      '%total' => $total,
+      '%title' => $solution->label(),
+    ]);
+    $solution->delete();
```

Comment:
If I save this snippet in PHP file and, with `develop` I run `drush scr file.php`, I can see that deleted solutions have, at least, child distributions. After running the update I see those distributions are orphans. Also I didn't check for group nodes, that can be easily done by inspecting the `node__og_audience` table. I think you should ask for clarifications from the functional team for underlying content should be cleaned as well.

Thinking out loud... _I would have been good to have string data integrity in place (referring to the ticket that we've been discussed for a long time)_

```php
<?php
$a = [
  // SCL
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=LST_NOM_DTL_LINEAR&StrNom=CL_ICD10&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=LST_NOM_DTL&StrNom=CL_MARSTA&StrLanguageCode=EN&IntPcKey=&StrLayoutCode=HIERARCHIC',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=LST_NOM_DTL&StrNom=CL_SEX&StrLanguageCode=EN&IntPcKey=&StrLayoutCode=HIERARCHIC',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=LST_NOM_DTL&StrNom=CL_ISCED97&StrLanguageCode=EN&IntPcKey=&StrLayoutCode=HIERARCHIC',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=LST_NOM_DTL&StrNom=CL_WSTATUS&StrLanguageCode=EN&IntPcKey=&StrLayoutCode=HIERARCHIC&IntCurrentPage=1',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=LST_NOM_DTL&StrNom=CL_IPC&StrLanguageCode=EN&IntPcKey=&StrLayoutCode=HIERARCHIC',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=LST_NOM_DTL&StrNom=CL_ICHA&StrLanguageCode=EN&IntPcKey=&StrLayoutCode=HIERARCHIC',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=LST_CLS_DLD&StrNom=CL_CURRENC&StrLanguageCode=EN&IntCurrentPage=1&StrLayoutCode=LINEAR',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ACCIDEN&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_AGE_&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_LEVELS&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_AMENITY&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_AQUAENV&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_AREA&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_BMI&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_CHANGE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_FIELD99&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_COFOG99&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_CPAYAGR&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_CURRENC&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_DAYSWEE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_DEG_URB&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_DISTANC&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ESTRUCT&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ESU&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ENGINE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_EC_DG&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ESOD03&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_FOS07&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_AREA_OC&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_FREQUEN&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_FRUITVA&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_GEO&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_HAZARD&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_FACILIT&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_HHCOMP&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_HOUSING&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_HHSTATU&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_LOC&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ICD9-CM&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_NCL&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ISCED11&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ISCO08&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ISCO88C&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_LCSTRUC&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ANIMALS&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_LSU&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_LOADSTA&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_MARSTA&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_MATURIT&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_MEAT&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_MEDIA&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_MED_SPE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ACCOMMO&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TRA_MOD&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_VALUE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_MONTH&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_NABS92&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_NABS07&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_ROOM_OC&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_NUMBER&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_NUTRIEN&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_OBZONE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_BODYPAR&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_PERSINV&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_RESID&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_DENSITY&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_PURPOSE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_QUANTIL&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_REQUEST&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_RBD&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_SEABASI&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TONNAGE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_SIZECLA&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_SIZE_HA&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_PERCENT&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_SMOKING&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_NST2007&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_SITC06&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_SITC88&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_STARTIM&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_NACE1&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_NACE2&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_CPA02&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_CPA08&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TAX&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TENURE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TERRTYP&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_T_FREQ&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TRAIN&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TRA_COV&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TRA_INF&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TREESPE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_trip_ar&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_BUILDIN&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_CARGO&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_CONSIGN&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TRA_OPE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_PLANTS&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_TRIPDES&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_BARK&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_VEHICLE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_WASTE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_CONSIST&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_WST_OPE&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_WEIGHT&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_WORKTIM&StrLanguageCode=EN',
  'http://ec.europa.eu/eurostat/ramon/nomenclatures/index.cfm?TargetUrl=ACT_OTH_DFLT_LAYOUT&StrNom=CL_Y_CONS&StrLanguageCode=EN',
  // Others.
  'http://ec.europa.eu/eurostat/ramon/miscellaneous/index.cfm?TargetUrl=DSP_GENINFO_SCL',
  'http://data.europa.eu/w21/35ef35c6-a530-4821-8c52-8d796ec86a1d',
  'http://data.europa.eu/w21/96f74e35-abda-4475-8b2d-22fed2120fc5',
];

foreach (\Drupal\rdf_entity\Entity\Rdf::loadMultiple($a) as $s) {
  print "Solution: {$s->id()}\n";
  if (!empty($s->toArray()['field_is_has_version'])) {
    print "Releases:\n";
    foreach ($s->toArray()['field_is_has_version'] as $r) {
      print "- {$r['target_id']}\n";
    }
  }
  if (!empty($s->toArray()['field_is_distribution'])) {
    print "Distributions:\n";
    foreach ($s->toArray()['field_is_distribution'] as $d) {
      print "- {$d['target_id']}\n";
    }
  }
  print "\n";
}

```